cli/src/commands/vkey.rs

```diff
@@ -0,0 +1,33 @@
+use std::fs::File;
+
+use anyhow::Result;
+use clap::Parser;
+use sp1_sdk::HashableKey;
+use sp1_sdk::ProverClient;
+use std::io::Read;
+
+#[derive(Parser)]
+#[command(name = "vkey", about = "View the verification key hash for a program.")]
+pub struct VkeyCmd {
+    /// Path to the ELF.
+    #[arg(long, required = true)]
+    elf: String,
+}
+
+impl VkeyCmd {
```

Comment:
will update template later to use this command in instructions and remove our custom bin to get the vkey hash